src/orders/urls.ts

```diff
@@ -78,6 +78,11 @@ export type OrderListUrlQueryParams = BulkAction &
   OrderListUrlSort &
   Pagination &
   ActiveTab;
+
+/**
+ * @deprecated
+ * This helper is likely broken, at least filters don't work. Either construct url manually or fix it
+ */
```

Comment:
it's not deprecated it's just that customer email filtering requires us to build filters from `ConditionalFilter` to create proper URL query

but we can still use that function to open modals, set active tab, change sorting, etc.: see. `OrderListUrlQueryParams`